### DISCUSSION TITLE
Submit button color

### DISCUSSION THREAD

**Discussion by dellydel** [dellydel/Intro-to-HTML-and-JS#22] [Q&A] [answered]
How do I change the color of a submit button?

**Comment by HD-primus** [comment, marked as answer]
Initiate an id tag inside the input tag for submit. Thereafter, add color/background color to the id tag in style.css by using #id

**Reply by BorisLaboratory** [reply]
I think the accepted answer need to be programmatic since we are dealing with codes. A step-wise algorithm or actual code spinet will be very helpful to others eg:

in side HTML file
==============

<input type"submit"  id="submit_button">


inside CSS file
==============

#submit_button{
  background-color: green;
  color: white;
}
